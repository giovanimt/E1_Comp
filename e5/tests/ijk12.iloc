addI rsp, 4 => rsp
loadI 2 => r0
loadI 4 => r1
loadI 2 => r2
div r1, r2 => r3
add r0, r3 => r4
loadI 4 => r5
loadI 4 => r6
mult r5, r6 => r7
add r4, r7 => r8
addI rfp, 0 => r9
store r8 => r9
