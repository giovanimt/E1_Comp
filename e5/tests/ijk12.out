00014180 20
